songhyun/app.js

```diff
@@ -42,6 +44,137 @@ app.post("/join", async (req, res) => {
     [name, password, email]
   );
   res.status(201).json({ message: "user created" });
+
+// 글 등록(과제3)
+app.post("/post", async (req, res) => {
+  const { title, content, user_id } = req.body;
+
+  await myDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      user_id
+    ) VALUES (?, ?, ?);
+    `,
+    [title, content, user_id]
+  );
+  res.status(201).json({ message: "postCreated" });
+});
+
+// 글 보기(과제4)
+app.get("/post/view", async (req, res) => {
```

Comment:
```suggestion
app.get("/posts", async (req, res) => {
```
view라는 키워드는 없어도 좋을 것 같습니다. 저희가 개발하는 API는 글 목록을 보여주기가 아니라 글 목록을 보여주기 위해 사용되는 `글 데이터`만 전달하는 것이기 때문입니다.

또한 method GET에 의미가 충분히 담겨 있으니 URI에는 자원 위주로 작성해주시면 좋을 것 같아요.